### ISSUE TITLE
client.watch() not a function?

### ISSUE THREAD

**Issue by drewfish** [mixer/etcd3#22]
Using etcd3@0.2.0 I'm trying the following:
```javascript
var etcd = require('etcd3');
var client = new etcd.Etcd3();
var watcher = client.watch().key(new Buffer('/')).create();
```
I get "TypeError: client.watch is not a function". Am I doing something wrong?

**Comment by drewfish** [comment]
Oh... I see that version `0.2.0` is not the head of master.

Poking around I was able to get this working:
```javascript
var etcd = require('etcd3');
var client = new etcd.Etcd3();
client.watch.watch().then((watcher) => {
    watcher.on('data', (data) => {
        data.events.forEach((evt) => {
            console.log(evt.type, evt.kv.key.toString());
        });
    });
    watcher.write({
        create_request: {
            key:        new Buffer('/'),
            range_end:  new Buffer('0'),
        }
    });
});
```

**Comment by connor4312** [comment]
Thanks for the issue! I published 0.2.1 which adds the watch usage as in `client.watch().key('/').create();`
